packages/components/src/styles/link-button-styles.ts

```diff
@@ -100,23 +115,26 @@ export const getLinkButtonStyles = (
       alignItems: 'flex-start',
       justifyContent: 'center',
       width: '100%',
-      minWidth: '54px', // ensure space is already reserved until icon component is loaded (ssr)
-      minHeight: '54px', // ensure space is already reserved until icon component is loaded (ssr)
+      ...(hideLabel && {
+        minWidth: minSizeHideLabel, // ensure space is already reserved until icon component is loaded (ssr)
+        minHeight: minSizeHideLabel, // ensure space is already reserved until icon component is loaded (ssr)
+      }),
```

Comment:
Is this the correct calculation? The rendered button/link has a computed min-width/min-height of 39px instead of 54px. If you delete the icon, you will see, that this value is not correct.